src/docs/releases/2.1.0.md

```diff
@@ -6,6 +6,8 @@ Release date: Not yet released
 
 ### External Authentication Feature
 
+#### External Authentication Logic Factored out from the Users Feature
```

Comment:
Maybe move the `### External Authentication Feature` to a new section **Added Features**

```suggestion
#### External authentication logic has been separated from the Users feature.
```

Reply:
This is a title, so should be capitalized like titles, and it's not a sentence thus shouldn't end with punctuation.

Reply:
fine. the suggestion was updated